addons/website_slides/controllers/main.py

```diff
@@ -525,6 +525,13 @@ def channel(self, channel=False, channel_id=False, category=None, category_id=Fa
                 }
```

Comment:
Commit header guidelines: Change "OptimizeSEO" to "website_slides" as it is the module we are modifying.